introduction.md

```diff
@@ -1,3 +1,8 @@
 I'm Vitor Navarro, Computer Science bachelor, software developer, gamer and book addict.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?